src/common/utils/localStorage.ts

```diff
@@ -0,0 +1,18 @@
+export const LocalStorage = (key: string) => {
```

Comment:
penamaan pascal case itu biasanya untuk nama class atau component.
kalo cuma variable/function, pake camel case

```suggestion
export const localStorage = (key: string) => {
```